src/ralph/backends/data/fs.py

```diff
@@ -0,0 +1,274 @@
+"""FileSystem data backend for Ralph."""
+
+import json
+import logging
+import os
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Generator, Iterable, Union
+from uuid import uuid4
+
+from ralph.backends.data.base import (
+    BaseDataBackend,
+    BaseDataBackendSettings,
+    BaseOperationType,
+    BaseQuery,
+    DataBackendStatus,
+    enforce_query_checks,
+)
+from ralph.backends.mixins import HistoryMixin
+from ralph.conf import BaseSettingsConfig, core_settings
+from ralph.exceptions import BackendParameterException
+from ralph.utils import now
+
+logger = logging.getLogger(__name__)
+
+
+class FileSystemDataBackendSettings(BaseDataBackendSettings):
+    """Represent the FileSystem data backend default configuration."""
+
+    class Config(BaseSettingsConfig):
+        """Pydantic Configuration."""
+
+        env_prefix = "RALPH_BACKENDS__DATA__FS__"
+
+    DEFAULT_DIRECTORY_PATH: str = str(core_settings.APP_DIR / "archives")
```

Comment:
Shouldn't it be the current directory instead?

Reply:
Good point) Thanks) Updated this part with `"."`)